docs/DEPLOYMENT.md

```diff
@@ -428,6 +429,19 @@ curl -X POST -H "Authorization: <access-token>" \
 ```
 Replace `<access-token>` with your JWT token. you must have `config-refresh` scope to call this API. 
 
+## Deploying on Disconnected Cluster
+
+- To deploy on a disconnected cluster you need to fork the [tektoncd/catalog][catalog] or you can use your own catalog but it must follow the [Catalog TEP][catalog-tep].
+- Fork the tektoncd/hub and update the [hub config][hub-config] file for your catalog details.
+- Mirror the Hub Images to the registry disconnected cluster has access to. You can use release images or build your own. you can use the latest release tag as the image tag.
```

Comment:
Could you please reframe this ??

Reply:
Would it be nice to point to the `Build Image` section of the deployment docs ??